materials/08_statistical_analysis_all_stages.Rmd

```diff
@@ -0,0 +1,417 @@
+---
+title: Statistical analysis of all 3 cell cycle stages
+bibliography: course_refs.bib
+editor_options: 
+  chunk_output_type: console
+editor: 
+  markdown: 
+    wrap: 72
+---
+
+::: callout-tip
+#### Learning Objectives
+
+-   Using the `limma` package (@Smyth2004), design a statistical model to test for
+    differentially abundant proteins between more than two conditions
+:::
+
+```{r, echo = FALSE, fig.align = "center", out.width = "90%"}
+knitr::include_graphics("figs/flow_chart/flow_chart.010.png", error = FALSE)
+```
+
+
+::: callout-note
+This workflow is an adjunct to the [Statistical analysis](./06_statistical_analysis.html) section which demonstrates how to perform single comparison statistical with `limma` (@Smyth2004). Please first read through that material, since it includes background explanations and clarifications which are not repeated here.
+:::
+
+```{r, eval=TRUE, include=FALSE}
+library("QFeatures")
+library("NormalyzerDE")
+library("limma")
+library("factoextra")
+library("org.Hs.eg.db")
+library("clusterProfiler")
+library("enrichplot")
+library("patchwork")
+library("tidyverse")
+library("pheatmap")
+library("here")
+
+load(here("materials/output/lesson04.rda"), verbose = TRUE)
+```
+
+## Differential expression analysis
+
+Having cleaned our data, aggregated from PSM to protein level, completed
+a log2 transformation and normalised the data, we are now ready to carry
+out statistical analysis. 
+
+The aim of this section is to answer the
+question: *"Which proteins show a significant change in abundance
+between the multiple cell cycle stages?"*.
+
+Our null hypothesis is: **(H0)** The change in abundance for a protein
+between cell cycle stages is zero.
+
+Our alternative hypothesis is: **(H1)** The change in abundance for a
+protein between cell cycle stages is not zero.
```

Comment:
I feel like this could be slightly more intuitive:

```suggestion
Our null hypothesis is: **(H0)** The mean protein abundance is the same across cell cycle stages.

Our alternative hypothesis is: **(H1)** The mean protein abundance differs across cell cycle stages.
```

It might be worth adding a note somewhere explicitly saying that the F test does not tell us which groups (cell cycle stages) are different from each other. Only that there is some difference due to cell cycle. However, we will later see how to extract pairwise comparisons between each cell cycle.

Reply:
Agreed on both points